### DISCUSSION TITLE
> pake-cli@2.3.6 tauri > tauri build ✶ Building app...        Error failed to get cargo metadata: No such file or directory (os error 2) file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:375                 reject(new Error(`${code}`));                        ^  Error: 1     at file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:375:24     at /usr/local/lib/node_modules/pake-cli/node_modules/shelljs/src/exec.js:145:9     at ChildProcess.exithandler (node:child_process:430:5)     at ChildProcess.emit (node:events:519:28)     at maybeClose (node:internal/child_process:1105:16)     at Socket.<anonymous> (node:internal/child_process:457:11)     at Socket.emit (node:events:519:28)     at Pipe.<anonymous> (node:net:337:12)

### DISCUSSION THREAD

**Discussion by shizhongyu** [tw93/Pake#648] [General]
> pake-cli@2.3.6 tauri
> tauri build
✶ Building app...
       Error failed to get cargo metadata: No such file or directory (os error 2)
file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:375
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:375:24
    at /usr/local/lib/node_modules/pake-cli/node_modules/shelljs/src/exec.js:145:9
    at ChildProcess.exithandler (node:child_process:430:5)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1105:16)
    at Socket.<anonymous> (node:internal/child_process:457:11)
    at Socket.emit (node:events:519:28)
    at Pipe.<anonymous> (node:net:337:12)

**Comment by tw93** [comment]
这个错误看不出问题，请放完整的错误
